app/swagger/requests/ppiu.rb

```diff
@@ -22,6 +22,37 @@ class PPIU
             end
           end
         end
+
+        operation :post do
+          extend Swagger::Responses::AuthenticationError
+
+          key :description, 'Update a veterans payment information'
+          key :operationId, 'postPaymentInformation'
+          key :tags, %w[form_526]
+
+          parameter :authorization
+
+          parameter do
+            key :name, :body
+            key :in, :body
+            key :description, 'Payment information to be updated for the user'
+            key :required, true
+
+            schema do
+              property :account_type, type: :string, example: 'Checking'
```

Comment:
Perhaps add the `required` key for each of the required properties, for the consumer's sake?